canonical/ubuntu.com
# Feature rebrand ceph

## Done

- Feature branch for ceph bubble rebrand

## QA

- View the site locally in your web browser at: https://ubuntu-com-14494.demos.haus/ceph
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check pages throughout bubble

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16340
